Parkinson/Resources/HTMLContent/AboutStudy.html

```diff
@@ -27,7 +27,7 @@
 
 We at Sage Bionetworks (nonprofit) want to understand why some people with PD have different symptoms than other people with PD, why a person's symptoms and side effects can vary over time and what can be done to help manage these symptoms day to day. <br /><br />
 
-Through surveys and phone sensor data, we can collect and track health and PD symptoms like dexterity, balance or gait. By participating in this study you will help us learn the range of PD symptoms and find out whether mobile devices can help measure PD progression and manage these symptoms better.  In this unique study, you will be a research partner. <br /><br />
+Through surveys and phone sensor data, we can collect and track health and PD symptoms like dexterity, balance and gait. By participating in this study you will help us learn the range of PD symptoms and find out whether mobile devices can help measure PD progression and manage these symptoms better.  In this unique study, you will be a research partner. <br /><br />
```

Comment:
OK done.  Thanks,
C

On Thu, Jul 30, 2015 at 2:56 PM, Erin-Mounts <notifications@github.com>
wrote:

> In Parkinson/Resources/HTMLContent/AboutStudy.html
> <https://github.com/Sage-Bionetworks/mPower/pull/41#discussion_r35925950>:
>
> > @@ -27,7 +27,7 @@
> >
> >  We at Sage Bionetworks (nonprofit) want to understand why some people with PD have different symptoms than other people with PD, why a person's symptoms and side effects can vary over time and what can be done to help manage these symptoms day to day. <br /><br />
> >
> > -Through surveys and phone sensor data, we can collect and track health and PD symptoms like dexterity, balance or gait. By participating in this study you will help us learn the range of PD symptoms and find out whether mobile devices can help measure PD progression and manage these symptoms better.  In this unique study, you will be a research partner. <br /><br />
> > +Through surveys and phone sensor data, we can collect and track health and PD symptoms like dexterity, balance and gait. By participating in this study you will help us learn the range of PD symptoms and find out whether mobile devices can help measure PD progression and manage these symptoms better.  In this unique study, you will be a research partner. <br /><br />
>
> oxford comma please ("...balance, and..."
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Sage-Bionetworks/mPower/pull/41/files#r35925950>.
>



-- 

Christine Suver  Ph.D., PMP
Principal Scientist, Head of Open Science Data Governance
*Sage Bionetworks* | 1100 Fairview Ave. North, M1-C108, Seattle, WA. 98109
(206) 667-2128 |  christine.suver@sagebase.org